app/lib/src/pages/landing_page.dart

```diff
@@ -2,14 +2,15 @@ import 'package:auto_route/auto_route.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:hooks_riverpod/hooks_riverpod.dart';
-import 'package:mondaymorning/src/services/themes/size_config.dart';
 import 'package:mondaymorning/src/services/router/mm_router.dart';
+import 'package:mondaymorning/src/services/themes/size_config.dart';
 import 'package:mondaymorning/src/store/states/landing_page/landing_page_data_type.dart';
 import 'package:mondaymorning/src/store/states/landing_page/landing_page_provider.dart';
 import 'package:mondaymorning/src/ui/components/home/landing_bottom_bar.dart';
 import 'package:mondaymorning/src/ui/screens/full_error_screen.dart';
 import 'package:mondaymorning/src/ui/screens/full_loading_screen.dart';
 
+@RoutePage<dynamic>(name: 'LandingRoute')
```

Comment:
Why dynamic? This page doesn't return any data so it should be void instead.

```suggestion
@RoutePage<void>(name: 'LandingRoute')
```